src/buildCommon.js

```diff
@@ -430,6 +430,20 @@ const makeFragment = function(
     return fragment;
 };
 
+/**
+ * Wraps group in a span if it's a document fragment, allowing to apply classes
+ * and styles
+ */
+const wrapFragment = function(
+    group: HtmlDomNode,
+    options: Options,
+): HtmlDomNode {
+    if (group instanceof DocumentFragment) {
+        return makeSpan([], [group], options);
```

Comment:
I guess this is equivalent to
```js
return makeSpan([], group.children, options);
```
? Not sure there are any advantages either way, though.